h/static/styles/core/_typography.scss

```diff
@@ -13,6 +13,9 @@ $mono-font-family: Open Sans Mono, Menlo, DejaVu Sans Mono, monospace !default;
 $normal-font-size: 13px;
 $normal-line-height: 15px;
 
+$big-font-size: 17px;
+$big-line-height: 22px;
```

Comment:
The text in flash messages needs to be bigger than normal to draw attention but there's no standard "big text" style in h. Copied these sizes from some random big text somewhere on activity pages